yarn.lock

```diff
@@ -5173,9 +5173,9 @@ slice-ansi@0.0.4:
   version "0.0.4"
   resolved "https://registry.yarnpkg.com/slice-ansi/-/slice-ansi-0.0.4.tgz#edbf8903f66f7ce2f8eafd6ceed65e264c831b35"
 
-"slickgrid@github:anthonydresser/SlickGrid#2.3.12":
-  version "2.3.12"
-  resolved "https://codeload.github.com/anthonydresser/SlickGrid/tar.gz/5610e05166cd7068dcf196498ae20054382b1684"
+"slickgrid@github:anthonydresser/SlickGrid#2.3.7":
```

Comment:
Did you intend to change this?

Reply:
Reverting the SlickGrid vbump is needed to fix the Restore Dialog.  @anthonydresser is the updated SlickGrid needed to fix any bugs in Feb.  Also, we vbumping again please verify Restore Dialog as part of that PR.

Reply:
I fixed the bug we had in restore, it looks like an npm problem...

Reply:
Restore Dialog was broke in master as of last night (at least on my machine after clean rebuild).  Reverting the updated slickgrid module fixed the issue.  If the issue is now resolved please update the package again if needed.  Thanks!